ibm_catalog.json

```diff
@@ -300,6 +300,202 @@
           ],
           "dependency_version_2": true,
           "terraform_version": "1.10.5"
+        },
+        {
+          "label": "Security-enforced",
+          "name": "security-enforced",
+          "install_type": "fullstack",
+          "working_directory": "solutions/security-enforced",
+          "compliance": {
+            "authority": "scc-v3",
+            "profiles": [
+              {
+                "profile_name": "CIS IBM Cloud Foundations Benchmark v1.1.0",
+                "profile_version": "1.1.0"
+              }
+            ]
+          },
+          "iam_permissions": [
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
+              "service_name": "kms",
+              "notes": "[Optional] Required if you are creating a new Key Protect Instance."
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "service_name": "iam-identity",
+              "notes": "[Optional] Required if Cloud automation for account configuration is enabled."
+            }
+          ],
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "existing_resource_group_name",
+              "display_name": "resource_group",
+              "required": false,
```

Comment:
Shouldn't this be required?

Reply:
No, as per the rally it was decided to not make it required as it always defaults to "Default". Also, it was advised to put this on top of optional variables.

Reply:
Not sure I agree with this - I think users are going to wan't to see the resource group in the required tab, as many users will not want to use the "Default" ?

Reply:
Agreed on slack we will keep in optional. There is no need to set `"required": false` - its already that way since it has a default value in terraform. Can we also re-order the inputs here so all required inputs are at the start, and then optional ones (region is required so should be above this for example)